### PR TITLE
Run autoformat

### PR DESCRIPTION
**Pull request by davidemerli**

### CODE REVIEW

File: text_calcio/cli/display.py [hunk 1]
```diff
@@ -283,7 +284,7 @@ async def penalty_interaction(
         print(self.display_header())
         print()
 
-        kick_direction_name, kick_direction_id = None, None
+        _kick_direction_name, kick_direction_id = None, None
```
Comment (dario-passarello) [inline text_calcio/cli/display.py:287]: Why ruff did this? I'm not a fan of undescore in front of variables

File: main.py [hunk 2]
```diff
@@ -1,9 +1,7 @@
-
-
 import asyncio
 import os
 import random
-from openai import AsyncOpenAI, OpenAI
+from openai import AsyncOpenAI
```
Comment (dario-passarello) [inline main.py:4]: Oh yes

File: text_calcio/cli/display.py (hunk 1 above)
Comment (davidemerli) [reply text_calcio/cli/display.py:287]: It throws an error due to the fact that their not being used, decided to underscore them myself (up to you what's the preferred style here)

Comment (dario-passarello) [reply text_calcio/cli/display.py:287]: Ok, I'm conviced

**Review by dario-passarello — APPROVED** [review]

**Pull request merged**